src/pipes/array/filter-by.ts

```diff
@@ -1,34 +1,53 @@
 import { Pipe, PipeTransform } from '@angular/core';
 import {
-  extractDeepPropertyByMapKey, extractDeepPropertyByParentMapKey, isBoolean, isNumberFinite, isString,
-  isUndefined
+  extractDeepPropertyByMapKey,
+  extractDeepPropertyByParentMapKey,
+  isBoolean,
+  isNumberFinite,
+  isString,
+  isUndefined,
 } from '../helpers/helpers';
 
 // tslint:disable no-bitwise
-@Pipe({name: 'filterBy'})
+@Pipe({ name: 'filterBy' })
 export class FilterByPipe implements PipeTransform {
-
-  transform(input: any[], props: Array<string>, search?: any, strict?: boolean): any[];
-  transform<T>(input: T, props: Array<string>, search?: any, strict?: boolean): T;
-  transform(input: any, props: Array<string>, search: any = '', strict: boolean = false): any {
-    if (!Array.isArray(input) || (!isString(search) && !isNumberFinite(search) && !isBoolean(search))) {
+  transform(
+    input: any[],
+    props: Array<string>,
+    search?: any,
+    strict?: boolean
+  ): any[];
+  transform<T>(
+    input: T,
+    props: Array<string>,
+    search?: any,
+    strict?: boolean
+  ): T;
+  transform(
+    input: any,
+    props: Array<string>,
+    search: any = '',
+    strict: boolean = false
+  ): any {
```

Comment:
No prettier is very opinionated by design and therefore has very few things that are configurable.

That said, I can set the line width to 120 if you prefer instead of 80.

Eventually clone my branch, change into `.prettierrc` file the `printWidth` from `80` to `120` and then run `npm run prettier:write`. Let me know what line width you prefer :+1: